### PR TITLE
feat: webhook registration apis

### PR DESCRIPTION
**Pull request by gulfamali**
PR contains:
1. register webhook app api
2. update webhook api
3. two additional keys in .env: `JWT_SECRET`, `WEBHOOK_SERVER_KEY`

NOTE: 
* Basic auth using WEBHOOK_SERVER_KEY as password is implemented for Register Api.
* JWT_SECRET is being used to generate account secret keys.

---
**Register API Curl Example**
```js
curl --location 'http://localhost:3000/register-app' \
--header 'Content-Type: application/json' \
--header 'Authorization: Basic d2ViaG9vazpqZGVkZzM3ZDM4ZDg=' \
--data-raw '{
    "name": "Pacman",
    "email": "gulfam1@priori.capital",
    "webhookUrl": "http://localhost:3000/health-check",
    "webhookAuthKey": "authkey12"
}'


/** 
* Here username is `webhook` and password is WEBHOOK_SERVER_KEY in .env. 
* For Basic Auth, generate string after concatenating username and password 
* separated by colon symbol and base64 encoded.
**/
btoa('webhook:jdedg37d38d8') => 'd2ViaG9vazpqZGVkZzM3ZDM4ZDg=';
```

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -68,6 +70,27 @@ export const start = async (chainId = 1): Promise<void> => {
     },
   });
 
+  server.auth.strategy("webhook_server", "basic", {
```
Comment (spunkypanda) [inline src/api/index.ts:73]: @gulfamali Can you attach a sample curl request with the auth header here, just as an example for understanding

File: package.json
Comment (spunkypanda) [inline package.json:1]: Add updated yarn.lock in future

**Review by spunkypanda — APPROVED** [review]
LGTM. Please attach a sample request curl as mentioned in one of the comments

File: src/api/index.ts (hunk 1 above)
Comment (gulfamali) [reply src/api/index.ts:73]:
```js
curl --location 'http://localhost:3000/register-app' \
--header 'Content-Type: application/json' \
--header 'Authorization: Basic d2ViaG9vazpqZGVkZzM3ZDM4ZDg=' \
--data-raw '{
    "name": "Pacman",
    "email": "gulfam1@priori.capital",
    "webhookUrl": "http://localhost:3000/health-check",
    "webhookAuthKey": "authkey12"
}'

/** 
* Here username is `webhook` and password is WEBHOOK_SERVER_KEY in .env. 
* For Basic Auth, generate string after concatenating username and password 
* separated by colon symbol and base64 encoded.
**/

btoa('webhook:jdedg37d38d8') => 'd2ViaG9vazpqZGVkZzM3ZDM4ZDg=';
```

**Pull request merged**